src/modules/iKinGazeCtrl/src/main.cpp

```diff
@@ -1225,18 +1233,38 @@ class GazeModule: public RFModule
         imuPort.setExchangeData(&commData);
         if (commData.stabilizationOn)
         {
-            imuPort.open(commData.localStemName+"/inertial:i");
-            if (Network::connect(remoteInertialName,imuPort.getName()))
-                yInfo("Receiving IMU data from %s",remoteInertialName.c_str());
-            else
-            {
-                yError("Unable to connect to %s!",remoteInertialName.c_str());
-                dispose();
-                return false;
+            if (commData.useMASClient) {
+                Property mas_conf{{"device", Value("multipleanalogsensorsclient")},
+                                  {"remote", Value("/"+commData.robotName+"/head/imurfe")},
```

Comment:
Could you please also take care of updating the documentation?

Reply:
Sure we have to settle the name of the ports, the first idea was `/robot_name/part/imu` or `imurfe` in case the robot mounts the robot facial expression board like the purple